### ISSUE TITLE
Random website glitching, Chrome 40, and some tweaking...

### ISSUE THREAD

**Issue by opennova** [liri-project/liri-browser#177]
Aloha,

I've been using Liri Browser for a while now, and I really do like it; it's like no other browser I've seen. I use this instead of Chrome; however, I am noticing some issues. Rather than creating three seperate tickets, I think it's more appropriate to list them all in one.

1. Websites tend to glitch out, like the image below.
![bug1](https://cloud.githubusercontent.com/assets/13445064/11648766/ca71324c-9d1b-11e5-99d8-558df02a4f35.png)

2. The Chromium version is 40, which is out of date and can't render certain websites, such as the new Google+ website.

3. The download circle appears to be way too big as compared to the arrow indicating download. Also, add to dash doesn't work, bookmarks should have folders, and there should be a 'clear history' option. Finally, there should be more HTML5 support as it reports 510/555 points.

Thank you for such a great browser, and i hope this feedback helps.

**Comment by pierremtb** [comment]
@opennova Thank you very much for this precise feedback.

1. and 2. (and the "finally" point) are QtWebEngine related, meaning they are upstream issues and we need to wait for Qt5.6 (we will switch to it as soon as the beta is out, should be Dec 10.)

3. I agree with this issue. However, can you create another issue for this one ? Because it's a different problem, design related.

**Comment by pierremtb** [comment]
Sorry didn't mean to close it.

**Comment by opennova** [comment]
Sure.

**Comment by ubunturox104** [comment]
I have been using version 0.3, the latest version.

**Comment by VirusKA** [comment]
2 more days till the QT 5.6 beta ;P

**Comment by tim-sueberkrueb** [comment]
@VirusKA yep \0/

**Comment by ubunturox104** [comment]
As it turns out, the download size and random glitching is coming just from the Linux version. The Mac version is working like a charm! ;)

**Comment by VirusKA** [comment]
@tim-sueberkrueb @pierremtb QT 5.6 beta was released last night!

**Comment by tim-sueberkrueb** [comment]
@VirusKA yes \0/ :)

**Comment by maxys92** [comment]
Is there a ETA for the New version?

**Comment by tim-sueberkrueb** [comment]
@maxys92 no, I'm afraid. We're currently quite busy with other things (life?) ;)
